permission_handler/example/lib/plugin_example/permission_list.dart

```diff
@@ -17,9 +17,10 @@ class PermissionList extends StatelessWidget {
                 if (Platform.isIOS) {
                   return permission != Permission.unknown &&
                       permission != Permission.sms &&
-                      //permission != Permission.storage &&
```

Comment:
shouldn't be commentary, [Here](https://stackoverflow.com/questions/53082568/how-can-i-request-ios-storage-permission-in-react-native) they also mention that iOS don't need/have a permission for storage.